### PR TITLE
Added F keys

### PR DESCRIPTION
**Pull request by Bram-Boris**
# Description

This PR adds the F keys to be used within the engine

# How can this be tested?

See cheats PR on BeastArena

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Included 1 or more screenshots
- [x] Code is const correct

### CODE REVIEW

File: include/brickengine/input.hpp [hunk 1]
```diff
@@ -398,6 +398,19 @@ class BrickInput {
         sdl_mapping.insert({InputKeyCode::Ekey_enter, SDLK_KP_ENTER});
         sdl_mapping.insert({InputKeyCode::EKey_backspace, SDLK_BACKSPACE});
 
+        sdl_mapping.insert({InputKeyCode::EKey_f1, SDLK_F1});
```
Comment (markJavab) [inline include/brickengine/input.hpp]: To follow the previous 'inserted' keys, I would add the comment // F keys above this section of inserted F keys.

File: include/brickengine/input.hpp [hunk 2]
```diff
@@ -461,6 +474,19 @@ class BrickInput {
         keycode_mapping.insert({SDLK_PAGEUP, InputKeyCode::EKey_pageup});
         keycode_mapping.insert({SDLK_HOME, InputKeyCode::EKey_home});
 
+        keycode_mapping.insert({SDLK_F1, InputKeyCode::EKey_f1});
```
Comment (markJavab) [inline include/brickengine/input.hpp]: To follow the previous 'inserted' keys, I would add the comment // F keys above this section of inserted F keys.

File: include/brickengine/input.hpp (hunk 1 above)
Comment (LuukSantegoeds) [reply include/brickengine/input.hpp]: I agree

File: include/brickengine/input.hpp (hunk 2 above)
Comment (LuukSantegoeds) [reply include/brickengine/input.hpp]: Also agree with this

**Pull request merged**